src/Concerns/UsesMultitenancyConfig.php

```diff
@@ -35,4 +36,9 @@ public function getMultitenancyActionClass(string $actionName, string $actionCla
 
         return app($configuredClass);
     }
+
+    public function getTenantArtisanSearchFields() : array
+    {
+        return Arr::wrap(config('multitenancy.tenant_artisan_search_fields') ?? 'id');
```

Comment:
Using `id` instead of `null` isn't necessary when `id` is set as the default in the config file.